packages/office-ui-fabric-react/src/components/List/List.tsx

```diff
@@ -306,10 +309,13 @@ export class List extends BaseComponent<IListProps, IListState> {
   }
 
   private _renderPage(page: IPage): any {
-    let { onRenderCell } = this.props;
+    let { onRenderCell, role } = this.props;
     let cells = [];
     let pageStyle = this._getPageStyle(page);
 
+    // only assign list item role if no role is assigned
+    role = role ? null : 'listitem';
```

Comment:
I wonder if we should provide a way to pass this in as a property.. but for now it's probably fine the way it is